validate_roles.py

```diff
@@ -3,15 +3,23 @@
 import json
 from termcolor import cprint
 
-REQUIRED_KEYS = {"role", "url", "sub-role", "lead", "deputy", "role-head",
-                 "role-subhead", "description", "sub-description"}
+REQUIRED_KEYS = {"role", "url", "role-head", "responsibilities"}
 
-LIST_KEYS = {"sub-role", "lead", "deputy", "role-head", "role-subhead",
-             "description", "sub-description"}
 
-STRING_KEYS = {"role", "url"}
+def assert_list(i, key, value):
```

Comment:
(very minor)
```suggestion
def assert_is_list(i, key, value):
```
(would also require updating where used below)